README.md

```diff
@@ -1,12 +1,23 @@
 # DF Template to Code
-This is a simple utility that will allow you to generate human readable pseudocode from a bunch of DiamondFire code templates.
-To use the utility:
-1. Make sure the required dependency is installed (org.json) in some way.
+This is a simple utility that will allow you to generate human-readable pseudocode from a bunch of DiamondFire code templates.
+
+## How to use
+### Building yourself
+1. Clone the repository
 2. Obtain the templates you want to convert
 3. Put them into a shulker box, then break it to get it as an item
 4. Hold the shulker and type `/i nbt`.
-5. Click the "Copy NBT" text.
-6. Paste it as a line in "templatedata.txt" (make sure to remove the placeholder text).
+5. Click the **"Click to copy unformatted NBT"** text.
+6. Paste it as a line in the "templatedata.txt" (make sure to remove the placeholder text).
```

Comment:
I was following the existing format, I'll change it

Reply:
I just re-read this, my bad.. `the` in step 6 refers to the existing txt file that is in the repository, since if you run the main method through IntelliJ it uses the root directory as the working directory, which means they should change the existing file and not make a new one somewhere unlike using a prebuilt jar